scripts/moderate_retrofitting.py

```diff
@@ -6,17 +6,17 @@
 warnings.filterwarnings("ignore")
 from plots._helpers import mock_snakemake, update_config_from_wildcards, load_network, \
                     change_path_to_pypsa_eur, change_path_to_base, load_unsolved_network, \
-                    save_unsolved_network
+                    save_unsolved_network, CO2L_LIMITS, LINE_LIMITS
 
 
 def set_moderate_retrofitting(n_solved, n_unsolved):
     # get optimal retrofitting from the solved network
     retro_opt = n_solved.generators.query("carrier in 'retrofitting'")[["p_nom_opt"]]
     retro_data = retro_opt / 2
-    # set p_nom as half of p_nom_opt of flexible scenario
-    n_unsolved.generators.loc[retro_data.index, "p_nom"] = retro_data["p_nom_opt"]
+    # set p_nom_max as half of p_nom_opt of flexible scenario
+    n_unsolved.generators.loc[retro_data.index, "p_nom_max"] = retro_data["p_nom_opt"].combine(n_unsolved.generators.loc[retro_data.index, "p_nom_min"], max)
     # set retrofitting not extendable
-    n_unsolved.generators.loc[retro_data.index, "p_nom_extendable"] = False
+    n_unsolved.generators.loc[retro_data.index, "p_nom_extendable"] = True
```

Comment:
... moderate retrofitting shouldn't have extendable retrofitting?
```suggestion
    n_unsolved.generators.loc[retro_data.index, "p_nom_extendable"] = False
```

Reply:
We set it so as you have suggested. We set `p_nom_max` as half of `p_nom_opt` of flexible and we allow to to be extended up to 1/2 p_nom_opt.